src/layout/Checkboxes/CheckboxesContainerComponent.tsx

```diff
@@ -80,21 +80,15 @@ const defaultOptions: IOption[] = [];
 const defaultSelectedOptions: string[] = [];
 
 export const CheckboxContainerComponent = ({
-  id,
```

Comment:
Well done with the props, looks much better to group them within the node prop. 👏